e2e/cypress/integration/enterprise/ldap/ldap_login_spec.js

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import users from '../../../fixtures/ldap_users.json';
+import {getRandomInt} from '../../../utils';
+
+// TODO: UPDATE THIS
+// assumes thet CYPRESS_* variables are set
+// assumes that E20 license is uploaded
+
+context('ldap', () => {
+    const user1 = users['test-1'];
+    const guest1 = users['board-1'];
+    const guest2 = users['board-2'];
+    const admin1 = users['dev-1'];
+
+    // const admin2 = users['dev-2'];
+    let testSettings;
+
+    const newConfig = {
+        ServiceSettings: {
+            SiteURL: Cypress.config('baseUrl'),
+        },
+        LdapSettings: {
+            Enable: true,
+            EnableSync: false,
+            LdapServer: Cypress.env('ldapServer'), // 'ldap.e2e.dev.spinmint.com' 'localhost'
+            LdapPort: Cypress.env('ldapPort'), //389,
+            ConnectionSecurity: '',
+            BaseDN: 'dc=mm,dc=test,dc=com',
+            BindUsername: 'cn=admin,dc=mm,dc=test,dc=com',
+            BindPassword: 'mostest',
+            UserFilter: '',
+            GroupFilter: '',
+            GuestFilter: '',
+            EnableAdminFilter: false,
+            AdminFilter: '',
+            GroupDisplayNameAttribute: 'cn',
+            GroupIdAttribute: 'entryUUID',
+            FirstNameAttribute: 'cn',
+            LastNameAttribute: 'sn',
+            EmailAttribute: 'mail',
+            UsernameAttribute: 'uid',
+            NicknameAttribute: 'cn',
+            IdAttribute: 'uid',
+            PositionAttribute: '',
+            LoginIdAttribute: 'uid',
+            SyncIntervalMinutes: 60,
+            SkipCertificateVerification: false,
+            QueryTimeout: 60,
+            MaxPageSize: 0,
+            LoginFieldName: '',
+            LoginButtonColor: '#0000',
+            LoginButtonBorderColor: '#2389D7',
+            LoginButtonTextColor: '#2389D7',
+            Trace: false
+        },
+        GuestAccountsSettings: {
+            Enable: true
+        },
+    };
+
+    describe('LDAP Login flow - Admin Login', () => {
+        before(() => {
+            // * Check if server has license for LDAP
+            cy.requireLicenseForFeature('LDAP');
+
+            cy.apiUpdateConfig(newConfig).then(() => {
```

Comment:
Please update to master.  In master, it needs to login explicitly in the spec file itself, like in this case as sysadmin (`cy.apiLogin('sysadmin')`) before doing API commands that require a certain permission.